judgels-backends/jophiel/jophiel-dist/Dockerfile

```diff
@@ -10,6 +10,6 @@ WORKDIR /judgels/jophiel
 EXPOSE 9001 9101
 
 COPY build/distributions .
+COPY init.sh .
```

Comment:
Instead of copying this in the Dockerfile, copy this in `scripts/deploy_jophiel.sh` into `build/distributions` instead, so there will be only a single atomic `COPY` in the Dockerfile

Reply:
Not yet?